pkgs/tools/misc/ectool/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "ectool";
+  version = "4.9";
+
+  src = fetchurl {
+    url = "https://coreboot.org/releases/coreboot-${version}.tar.xz";
+    sha256 = "0xkai65d3z9fivwscbkm7ndcw2p9g794xz8fwdv979w77n5qsdij";
+  };
+
+  buildPhase = ''
+    make -C util/ectool
```

Comment:
How about?
```
makeFlags = [ 
  "PREFIX=${placeholder "out"}"
  "INSTALL=install"
];

preBuild = ''
  cd util/ectool
'';
```

Then I think you can drop the custom `installPhase`, though upstream should really be using `DESTDIR` too
> https://github.com/coreboot/coreboot/blob/master/util/ectool/Makefile#L38

Reply:
I would like to stick to the pattern used in all other coreboot util packages for now, and refactor or merge them later.